Improve documentation of template modifiers (#380)
WoltLab/docs.woltlab.com: docs/view/templates.md

#### Modifiers

Variable modifiers are used to modify the output of variables within templates.
These modifiers allow you to perform various operations on the variables before displaying them.
The general syntax for applying a variable modifier is `{$variable|modifier}`.

Modifiers can be chained together to perform multiple operations on a variable. In such cases, the modifiers are applied from left to right. For example:
```smarty
{$variable|modifier1|modifier2|modifier3}
```

A modifier may accept additional parameters that affect its behavior. These parameters follow the modifier name and are separated by a `:`. For example:
```smarty
{$variable|modifier:'param1':'param2'}
```

> An overview of all available modifiers can be found [here](template-modifiers.md).

Reviewer: Avoid “here” as link text:
```suggestion
An [overview of all available modifiers](template-modifiers.md) is available on a [dedicated page](template-modifiers.md).
```
